controller/api/public/grpc_server_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes/duration"
-	"github.com/linkerd/linkerd2/controller/api/proxy"
+	"github.com/linkerd/linkerd2/controller/api/destination"
```

Comment:
Oof, why does this test depend on the proxy package? Looks like this dependency was introduced in #2195, when we added the `linkerd endpoints` command. My preference would be to move it to a separate discovery package and have both destination and public depend on that. But I'm happy to do that in a follow-up branch. In the meantime, once you change the destination package name as suggested above, you'll need to use the new package name in this file, like so:

```diff
diff --git a/controller/api/public/grpc_server_test.go b/controller/api/public/grpc_server_test.go
index f30030cc..cf3c8b6c 100644
--- a/controller/api/public/grpc_server_test.go
+++ b/controller/api/public/grpc_server_test.go
@@ -549,7 +549,7 @@ func TestEndpoints(t *testing.T) {
 			}
 			k8sAPI.Sync()
 
-			discoveryClient, gRPCServer, proxyAPIConn := proxy.InitFakeDiscoveryServer(t, k8sAPI)
+			discoveryClient, gRPCServer, proxyAPIConn := destination.InitFakeDiscoveryServer(t, k8sAPI)
 			defer gRPCServer.GracefulStop()
 			defer proxyAPIConn.Close()
 
```